graspologic/cluster/rclust.py

```diff
@@ -0,0 +1,320 @@
+# Copyright (c) Microsoft Corporation and contributors.
```

Comment:
Can we rename this file `recursive_cluster.py`?  Makes it easier to find it later if we name it that way off the top :)

Reply:
we were debating renaming the class `DivisiveCluster` I think there is a comment about that somewhere, but happy to have the file name be more descriptive

Reply:
@tliu68 can you make this change? And maybe we go with DivisiveCluster for now? Or DivisiveHierarchicalCluster if we wanna be overly verbose.

Reply:
Will do!

Reply:
`divisive_cluster.py` and `DivisiveCluster` sound like fine names to me!